TODO/todo_app/models.py

```diff
@@ -0,0 +1,41 @@
+from django.utils import timezone
+
+from django.db import models
+from django.urls import reverse
+
+# to add a default due time while creating a todo item
+def one_week_hence():
+    return timezone.now() + timezone.timedelta(days=7)
+
+#defining title of the list
+class ToDoList(models.Model):
+    title = models.CharField(max_length=100, unique=True)
+
+    def get_absolute_url(self):
```

Comment:
What will this function do?

Reply:
get_absolute_url(): Tells the django how to calculate the canonical URL for an object, returns a string that can be used to refer to an object over the HTTP.
for example:
def get_absolute_url(self):
        return reverse("list", args=[self.id])
Here,
      this function returns the URL for the list item refering to the id of the object.